ProvisionSample/Program.cs

```diff
@@ -328,7 +329,32 @@ static void Main(string[] args)
                         Console.WriteLine("Press any key to exit..");
                         exit = true;
                         break;
-                }
+                }
+            }
+            catch (HttpOperationException ex)
+            {
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.WriteLine("Ooops, something broke: {0}", ex.Message);
+                var error = SafeJsonConvert.DeserializeObject<PBIExceptionBody>(ex.Response.Content);
+                if (error != null && error.Error != null)
+                {
+                    if (error.Error.Details != null && error.Error.Details.FirstOrDefault() != null)

```

Comment:
error.Error.Details.FirstOrDefault() is called twice.
save it in paramter inside the if and check its value.